src/app/portlets/dot-edit-page/content/services/html/iframe-edit-mode.js.ts

```diff
@@ -32,32 +32,29 @@ export const EDIT_PAGE_JS = `
         return model;
     }
 
-    function checkIfContentletTypeIsAcepted(el, target){
+    function checkIfContentletTypeIsAccepted(el, target) {
         return el.dataset.dotBasetype === 'WIDGET' ||
                el.dataset.dotBasetype === 'FORM' ||
                target.dataset.dotAcceptTypes.indexOf(el.dataset.dotType) > -1;
     }
 
-    function checkIfMaxLimitNotReached(target){
+    function checkIfMaxLimitNotReached(target) {
         return target.children.length < parseInt(target.dataset.maxContentlets, 10)
     }
 
-    function checkIfContentletIsUnique(el, target){
-        return !Array.from(target.querySelectorAll("div[data-dot-object='contentlet']"))
-            .filter(node => {
-                return (node.dataset.dotInode === el.dataset.dotInode ||
-                        node.dataset.dotIdentifier === el.dataset.dotIdentifier)
-                }).length;
+    function checkIfContentletIsUnique(el, target) {
```

Comment:
@hmoreras D&D wasn't working because of this check, so what I did was I create an array of inodes from the target containers and compare with that, seems easy to me.